tests/components/zha/test_switch.py

```diff
@@ -507,3 +562,125 @@ async def test_switch_configurable(
 
     # test joining a new switch to the network and HA
     await async_test_rejoin(hass, zigpy_device_tuya, [cluster], (0,))
+
+
+WCAttrs = closures.WindowCovering.AttributeDefs
+WCT = closures.WindowCovering.WindowCoveringType
+WCCS = closures.WindowCovering.ConfigStatus
+WCM = closures.WindowCovering.WindowCoveringMode
+
+
+async def test_cover_inversion_switch(
+    hass: HomeAssistant, zha_device_joined_restored, zigpy_cover_device
+) -> None:
+    """Test ZHA cover platform."""
+
+    # load up cover domain
+    cluster = zigpy_cover_device.endpoints.get(1).window_covering
```

Comment:
minor, but inconsistent in the tests, we can probably just use `endpoints[1]` everywhere.